### DISCUSSION TITLE
May I use Bluetooth connection?

### DISCUSSION THREAD

**Discussion by starmisszhang** [TeamOpenFIRE/OpenFIRE-Firmware#38] [General]
Pico W  Can I directly connect to my computer using Bluetooth? I bought one and burned the corresponding firmware, but I don't know how to use it

**Comment by SeongGino** [comment]
The Pico W firmware images provided in releases don't have Bluetooth support.

It is for debugging and development purposes only, which for prospective developers [should be pretty obvious](https://github.com/TeamOpenFIRE/OpenFIRE-Firmware/issues/22)
